app/models/ability.rb

```diff
@@ -5,7 +5,7 @@ def initialize(user)
     user.blank? ? return : (@user = user)
     department_head_abilities if user.is_department_head?
     employee_abilities if user.is_employee?
-    resolver_abilities if user.is_resolver?
+    # resolver_abilities if user.is_resolver?
```

Comment:
Also comment `resolver_abilities` method for less ambiguity.

Reply:
commented